package.json

```diff
@@ -9,6 +9,9 @@
 		"email": "sindresorhus@gmail.com",
 		"url": "sindresorhus.com"
 	},
+  "contributors": [
+    "Mark Stosberg <mark@rideamigos.com>"
+  ],
```

Comment:
I don't use this field. It's too much work to maintain and it's not used for anything.

Reply:
It's used to provide credit in package.json to people who are not you.

Reply:
As an open source maintainer myself, my experience has been contributors do care about credit and it's basically free to give it to them. They get no money in return and this is something I can gave them in exchange for their work that has may have some value to them but costs me only a moment of time. 

If someone were add themselves as a contributor I'd accept the patch. If they don't, then they didn't care that much about it and it's no work for me.

Reply:
The commit history shows everyone's contributions. So does https://github.com/sindresorhus/crypto-random-string/graphs/contributors. And these don't favor only the people that know about the `contributors` field.